dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -405,6 +405,48 @@ public void ReorderItemsTest()
             }
         }
 
+        [TestMethod]
+        public void ItemChangedEventOnDragTest()
+        {
+            if (PlatformConfiguration.IsOSVersionLessThan(OSVersion.Redstone5))
+            {
+                // TODO 19727004: Re-enable this on versions below RS5 after fixing the bug where mouse click-and-drag doesn't work.
+                Log.Warning("This test relies on touch input, the injection of which is only supported in RS5 and up. Test is disabled.");
+                return;
+            }
+
+            using (var setup = new TestSetupHelper("TabView Tests"))
+            {
+                Button addButton = FindElement.ByName<Button>("Add New Tab");
+                Verify.IsNotNull(addButton);
+
+                Log.Comment("Add tab so scroll buttons appear.");
+                addButton.InvokeAndWait();
+
+                Verify.IsTrue(AreScrollButtonsVisible(), "Scroll buttons should appear");
+
+                UIObject sourceTab =  FindElement.ByName("FirstTab");
+
+                Verify.IsNotNull(sourceTab);
+
+                UIObject dropTab = FindElement.ByName("LastTab");
+                Verify.IsNotNull(dropTab);
+
+                Log.Comment("Dragging tab to the last overflow tab...");
+                InputHelper.DragToTarget(sourceTab, dropTab);
```

Comment:
@kmahone I can't seem to get the drag to work locally to verify the test. Any ideas?

Reply:
discussed offline. Runnint the tests via VM solved the issue.